code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -167,20 +167,20 @@
 	data["isPillBottleLoaded"] = bottle ? 1 : 0
 	if(bottle)
 		var/datum/component/storage/STRB = bottle.GetComponent(/datum/component/storage)
-		data["pillBotContent"] = bottle.contents.len
-		data["pillBotMaxContent"] = STRB.max_items
+		data["pillBottleCurrentAmount"] = bottle.contents.len
+		data["pillBottleMaxAmount"] = STRB.max_items
 
 	var/beakerContents[0]
 	if(beaker)
 		for(var/datum/reagent/R in beaker.reagents.reagent_list)
 			beakerContents.Add(list(list("name" = R.name, "id" = ckey(R.name), "volume" = R.volume))) // list in a list because Byond merges the first list...
-		data["beakerContents"] = beakerContents
+	data["beakerContents"] = beakerContents
```

Comment:
because with the recent change in the way tgui merges static data, we need for this array to stay, otherwise UI will retain the last known value in the beaker.

Reply:
I see.